.changeset/big-points-try.md

```diff
@@ -1,5 +1,4 @@
 ---
-"@ignored/hardhat-vnext-core": major
```

Comment:
Each of theses files is the record of a previous release of a `pre-release`, in effect they allow changeset to record some ordering information on the prereleases that isn't encoded in npm.
I have removed the entries for core for the historical record for simplicity.